src/main/java/org/snomed/snowstorm/core/data/services/AxiomConversionService.java

```diff
@@ -61,11 +61,25 @@ public void populateAxiomMembers(Collection<Concept> concepts, String branchPath
 		AxiomRelationshipConversionService conversionService = setupConversionService(branchPath);
 		for (Concept concept : concepts) {
 			for (Axiom axiom : concept.getClassAxioms()) {
-				String owlExpression = conversionService.convertRelationshipsToAxiom(mapFromInternalRelationshipType(concept.getConceptId(), axiom.getDefinitionStatusId(), axiom.getRelationships(), true));
+				String owlExpression;
+				ReferenceSetMember referenceSetMember = axiom.getReferenceSetMember();
+				if (!axiom.isActive() && referenceSetMember != null) {
+					owlExpression = referenceSetMember.getAdditionalField(ReferenceSetMember.OwlExpressionFields.OWL_EXPRESSION);
+				} else {
+					owlExpression = conversionService.convertRelationshipsToAxiom(mapFromInternalRelationshipType(concept.getConceptId(), axiom.getDefinitionStatusId(), axiom.getRelationships(), true));
```

Comment:
@pgwilliams The problem comes from the **convertRelationshipsToAxiom** method. Whenever a Concept is being updated, it's Axioms are being re-generated. I've changed this so the Axioms are only re-generated if they are active, otherwise use what was there previously.